src/04-modality-specific-files/05-task-events.md

```diff
@@ -131,14 +145,65 @@ of example.
        "Description": "ID from KDEF database used to identify the displayed image"
    }
 }
-```	   
+```
 
-For multi-echo files, the `*_events.tsv` file is applicable to all echos of particular
-run:
+Note that all other columns SHOULD also be described but are omitted for the
+sake of example.
 
-```Text
-sub-01_task-cuedSGT_run-1_events.tsv
-sub-01_task-cuedSGT_run-1_echo-1_bold.nii.gz
-sub-01_task-cuedSGT_run-1_echo-2_bold.nii.gz
-sub-01_task-cuedSGT_run-1_echo-3_bold.nii.gz
+## Stimulus presentation details
+
+Specific fields related to stimulus presentation details SHOULD also be present
+in the accompanying JSON sidecar:
+
+| Field name           | Description                                                                                                                                                                                                                                                    |
+|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| StimulusPresentation | RECOMMENDED. Object containing key value pairs related to the software used to present the stimuli during the experiment, specifically: `OperatingSystem`, `SoftwareName`, `SoftwareRRID`, `SoftwareVersion` and `Code`. See table below for more information. |
```

Comment:
making a link to JSON objects, like in this PR: https://github.com/bids-standard/bids-specification/pull/587/files

at the end of the file, you'll need to insert:

```Text
<!-- Link Definitions -->

[object]: https://www.json.org/json-en.html
```

... if there are already other links at the end of the file, just merge all links into a single such "comment-separated-section"

```suggestion
| StimulusPresentation | RECOMMENDED. [Object][] containing key value pairs related to the software used to present the stimuli during the experiment, specifically: `OperatingSystem`, `SoftwareName`, `SoftwareRRID`, `SoftwareVersion` and `Code`. See table below for more information. |
```

Reply:
Just to know: is that case sensitive?

Will the following work?

```
[Object][]

<!-- Link Definitions -->

[object]: https://www.json.org/json-en.html
```

Or should it be?

```
[Object][]

<!-- Link Definitions -->

[Object]: https://www.json.org/json-en.html
```

Reply:
it should work, AFAIK links in md work like:

- make everything lowercase
- strip all punctuation
- replace whitespace with `-`

and in the PR I mentioned it also works :)

Reply:
This should be done then. :smile: